modules/network/src/integrationTest/java/org/apache/ignite/internal/network/netty/ItConnectionManagerTest.java

```diff
@@ -302,57 +311,56 @@ public void testStopTwice() throws Exception {
 
     @Test
     public void testOneConnectionType() throws Exception {
-        int size = 10000000;
-        char[] chars = new char[size];
-        Arrays.fill(chars, 'a');
-        String bigText = new String(chars);
+        String bigText = IgniteTestUtils.randomString(new Random(), 10000000);
 
+        try (ConnectionManagerWrapper server1 = startManager(4000);
+                ConnectionManagerWrapper server2 = startManager(4001)) {
+            NettySender sender = server1.openChannelTo(server2).get(3, TimeUnit.SECONDS);
 
-        ConnectionManagerWrapper server1 = startManager(4000);
-        ConnectionManagerWrapper server2 = startManager(4001);
+            TestMessage bigMessage = messageFactory.testMessage().msg(bigText).build();
+            TestMessage msg = messageFactory.testMessage().msg("test").build();
 
-        NettySender sender = server1.openChannelTo(server2, ChannelType.DEFAULT).get(3, TimeUnit.SECONDS);
+            CompletableFuture<Void> send1 = sender.send(new OutNetworkObject(bigMessage, Collections.emptyList()));
+            CompletableFuture<Void> send2 = sender.send(new OutNetworkObject(msg, Collections.emptyList()));
 
-
-        TestMessage bigMessage = messageFactory.testMessage().msg(bigText).build();
-        TestMessage msg = messageFactory.testMessage().msg("test").build();
-
-        CompletableFuture<Void> send1 = sender.send(new OutNetworkObject(bigMessage, Collections.emptyList()));
-        CompletableFuture<Void> send2 = sender.send(new OutNetworkObject(msg, Collections.emptyList()));
-
-        assertThat(send2, willCompleteSuccessfully());
-        assertThat(send1.isDone(), is(true));
-
-        server1.close();
-        server2.close();
+            assertThat(send2, willCompleteSuccessfully());
+            assertTrue(send1.isDone());
+        }
     }
 
     @Test
     public void testTwoConnectionTypes() throws Exception {
-        int size = 1000000000;
-        char[] chars = new char[size];
-        Arrays.fill(chars, 'a');
-        String bigText = new String(chars);
-
+        String bigText = IgniteTestUtils.randomString(new Random(), 100000000);
```

Comment:
I don't really like that we have a 100Mb message in test. Can you please document the test scenario, and justify why it has to be this big?